package-lock.json

```diff
@@ -1,1919 +1,8 @@
 {
```

Comment:
please update your npm version or don't commit this file if you haven't installed any new packages.